test/unit/org/apache/cassandra/cql3/ast/Select.java

```diff
@@ -303,6 +303,7 @@ public OrderBy build()
 
     public static class Builder implements Conditional.ConditionalBuilder<Builder>
     {
+        private boolean filtering = false;
```

Comment:
you can remove this.  the ast rewrite fixed a style issue with this field, the fields try to use the same thing as the CQL so became `allowFiltering`, and the logic from harry v2 just got renamed (its why this is dead code)